.codesandbox/ci.json

```diff
@@ -11,7 +11,8 @@
     "/examples/starter-algolia",
     "/examples/starter",
     "/examples/voice-search",
-    "/examples/vue"
+    "/examples/vue",
+    "/examples/tags-with-hits"
```

Comment:
I think there's a limited number of examples

Reply:
See [PR description](https://github.com/algolia/autocomplete/pull/646#issue-705069124):

>I've temporarily added the sandbox to CodeSandbox CI so that it builds the new plugin and adds it as a dependency, otherwise we can't visualize it. We can remove it before we merge.